index.html

```diff
@@ -11,31 +11,27 @@
     href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
     integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog=="
     crossorigin="anonymous" />
+  <link href="assets/css/bootstrap-responsive.css" rel="stylesheet">
   <title>Student Grade Table</title>
 </head>
 <body>
   <header>
-    <div class="container page-header p-0">
-      <div class="row justify-content-between align-items-center">
-        <div class="col-8">
+    <div class="container page-header ">
+      <div class="row ">
+        <div class="col-md-8 text-center text-md-left">
```

Comment:
Make sure to have an un-sized `col` class as well, otherwise it won't format as nicely on small devices (i.e., use `col col-md-8` instead of just `col-md-8`).

Reply:
@ArlletteFonseca To clarify, by "sized" I mean "for a sub-set of device sizes", which are the classes with `*-sm-*`, `*-md-*`, `*-lg-*`, `*-xl-*`.